woocommerce/woocommerce
# Fixes #22559

### All Submissions:

 

 
### Changes proposed in this Pull Request:
Changes in admin.css and admin-rtl.css. 
Fixes #22559 

### How to test the changes in this Pull Request:

1. Admin > Users > Edit User 
2. Select Country 
3. State / county field, and check select container and description is too close:

### Other information:
 
- [x]  Have you written new tests for your changes, as applicable?[x]  Have you successfully ran tests with your changes locally?

### Changelog entry 
Fix - In edit customer screen, state / county field, select container and description is too close.
